tox.ini

```diff
@@ -22,15 +22,18 @@ setenv =
   PYTHONPATH={toxinidir}/src:{toxinidir}/lib:{toxinidir}/image-builder/src:{toxinidir}/image-builder/files
   TZ=UTC
 
-[testenv:integration]
-passenv =
-  HOME
-  JUJU_REPOSITORY
-  PATH
-  WORKSPACE
+[testenv:test-build]
```

Comment:
Interesting, potentially worth adding to other charms as well, e.g., indico, any thoughts @arturo-seijas/ @mthaddon?

Reply:
The problem here is that this is coupled with microk8s (because of the tag), so changing the repository will imply a change here as well.
Anyway, we'd need to first test GitHub's container registry performance. We have a ticket in the backlog for that.

Reply:
Just noticed this is for local dev